code/script.js

```diff
@@ -1,9 +1,35 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+
+const acc = document.getElementById("section1")
+const pru = document.getElementById("section2")
+const lan = document.getElementById("section3")
+const val = document.getElementById("section4")
```

Comment:
I would work on some better variables names. They should be able to stand by themself and you should know what they mean. I wouldn't know what `lan` or `pru` stands for example